examples/Huggingface_Transformers/README.md

```diff
@@ -203,6 +205,7 @@ Change `setup_config.json` to
  "save_mode":"pretrained",
  "max_length":"150",
  "captum_explanation":true,
+ "FasterTransformer":false,
```

Comment:
the key naming convention is different from others. Should we make it consistent?
And do we need explicitly set to false?

Reply:
This has been set to represent the FasterTrasnfromer from Nvidia, it should be set to false, this has been added to support FasterTransformer example.